versioned_docs/version-8.6/apis-tools/camunda-api-rest/specifications/sidebar.js

```diff
@@ -5,235 +5,205 @@ module.exports = [
   },
   {
     type: "category",
-    label: "Cluster",
+    label: "Clock",
     items: [
       {
         type: "doc",
-        id: "apis-tools/camunda-api-rest/specifications/get-cluster-topology",
-        label: "Get cluster topology",
-        className: "api-method get",
+        id: "apis-tools/camunda-api-rest/specifications/pin-internal-clock-alpha",
```

Comment:
non-blocking, no action requested, informative comment: 

Thanks for doing this reordering!

You might be wondering why you had to do it in two places (here, and in the top-level version-8.6-sidbars.json). As tracked in https://github.com/camunda/camunda-docs/issues/3027, this particular sidebar.js file is unused by docusaurus, because numbered versions use a static `.json` file for the top-level sidebars. This file only exists in the 8.6 docs because docusaurus copied it over during versioning, and we never deleted it.

All that to say, any changes in this file will just be ignored. If anything, we should probably delete this file....but that is not your responsibility.